src/components/members/detail/GroupSection/index.tsx

```
@@ -22,22 +23,30 @@ interface GroupSectionProps {
     return <EmptyProfile />;
   }
+  const meetingList = meetingData?.userAppliedMeetings ?? []; // 데이터를 안전하게 추출
+  const ITEMS_PER_ROW = 2;
+  const ROW_COUNT = Math.ceil(meetingList.length / ITEMS_PER_ROW);
+
+  const rowVirtualizer = useVirtualizer({
+    count: ROW_COUNT,
+    getScrollElement: () => parentRef.current,
+    overscan: 6,
+    estimateSize: useCallback(() => 136, []),
+  });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)